domain/src/main/java/org/oppia/android/domain/profile/ProfileManagementController.kt

```diff
@@ -721,8 +721,29 @@ class ProfileManagementController @Inject constructor(
     return imageFile.absolutePath
   }
 
-  private fun onlyLetters(name: String): Boolean {
-    return name.matches(Regex("^[ A-Za-z]+\$"))
+  private fun nameAllowed(name: String): Boolean {
```

Comment:
Given that we're considering also adding a bad word filter in the future, it might be prudent to create a new NameValidator class in the utility module and have name logic added there (along with dedicated tests so that you can more easily cover edge cases vs. having to test those through the profile management controller).

Reply:
> along with dedicated tests so that you can more easily cover edge cases vs. having to test those through the profile management controller

Interesting! Should I also move the tests related to allowed and unallowed names from profile management controller's test to name validator class's test?

Reply:
I think you should focus on adding new test cases that are separate from the ProfileManagementController's tests (you also get redundancy this way).